src/sdk/conference/signaling.js

```diff
@@ -149,4 +151,43 @@ export class SioSignaling extends EventModule.EventDispatcher {
       });
     });
   }
+
+  /**
+   * @function _onReconnectionTicket
+   * @instance
+   * @desc Parse reconnection ticket and schedule ticket refreshing.
+   * @memberof Oms.Conference.SioSignaling
+   * @private.
+   */
+  _onReconnectionTicket(ticketString) {
+    this._reconnectionTicket = ticketString;
+    const ticket = JSON.parse(Base64.decodeBase64(ticketString));
```

Comment:
validate the string first?

Reply:
This string is issued by server. If there is an invalid input, simply throw an exception here and abort the process. Nothing we can do to correct this string.